src/Windows/Avalonia.Win32/Win32NativeControlHost.cs

```diff
@@ -128,7 +128,7 @@ class Win32NativeControlAttachment : INativeControlHostControlTopLevelAttachment
         {
             private DumbWindow _holder;
```

Comment:
minor: ```DumbWindow _holder``` can be marked nullable, and CheckDisposed can has MemberNotNull atrribute.

Reply:
I would put "_child" there too